jira_assignee_from_calendar/autokitteh.yaml

```diff
@@ -16,7 +16,7 @@ project:
   name: jira_assignee_from_calendar
   vars:
     - name: SHARED_CALENDAR_ID
-      value: primary
+      value: 
```

Comment:
If "primary" is the API's default, I'd keep it as a default here too.

Reply:
Done